app/src/main/java/com/spendster/presentation/MyViewPagerAdapter.java

```diff
@@ -0,0 +1,42 @@
+package com.spendster.presentation;
+
+import android.content.Context;
+import android.support.v4.view.PagerAdapter;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+public class MyViewPagerAdapter extends PagerAdapter {
+    private LayoutInflater layoutInflater;
+    private int[] layouts;
+    private WelcomeActivity welcomeActivity;
```

Comment:
@SimonMaha Activity should not be passed as parameters, see memory leaks with passing Activity refs.